app/models/ticket.js

```diff
@@ -44,5 +44,13 @@ export default ModelBase.extend({
 
   itemTotal: computed('price', 'quantity', function() {
     return this.price * this.quantity;
+  }),
+
+  ticketPriceWithTax: computed('event.tax', function() {
+    let taxType = this.event.get('tax.isTaxIncludedInPrice');
+    if (!taxType && taxType !== undefined) {
```

Comment:
Can you elaborate on this logical comparison?

Reply:
I have rewritten it to use only `taxType !== undefined` just in case if there is no tax object in connection to the event.

Also, I am now using `computed('event'....)` . I have to use `event.get('tax')` because `event` is in relationship with the tickets and to access their attributes, I have to resolve the promise in a model. @CosmicCoder96